acl/types.go

```diff
@@ -12,4 +24,73 @@ const (
 
 	// PublicAppendRule is a basic ACL value for public-append container.
 	PublicAppendRule = 0x1FBF9FFF
+
+	// EACLPublicBasicRule is a basic ACL value for eacl-public-read-write container.
+	EACLPublicBasicRule = 0x0FBFBFFF
+
+	// EACLPrivateBasicRule is a basic ACL value for eacl-private container.
+	EACLPrivateBasicRule = 0x0C8C8CCC
+
+	// EACLReadOnlyBasicRule is a basic ACL value for eacl-public-read container.
+	EACLReadOnlyBasicRule = 0x0FBF8CFF
+
+	// EACLPublicAppendRule is a basic ACL value for eacl-public-append container.
+	EACLPublicAppendRule = 0x0FBF9FFF
 )
+
+const (
+	// PublicBasicName is a well-known name for 0x1FBFBFFF basic ACL.
+	PublicBasicName = "public-read-write"
+
+	// PrivateBasicName is a well-known name for 0x1C8C8CCC basic ACL.
+	PrivateBasicName = "private"
+
+	// ReadOnlyBasicName is a well-known name for 0x1FBF8CFF basic ACL.
+	ReadOnlyBasicName = "public-read"
+
+	// PublicAppendName is a well-known name for 0x1FBF9FFF basic ACL.
+	PublicAppendName = "public-append"
+
+	// EACLPublicBasicName is a well-known name for 0x0FBFBFFF basic ACL.
+	EACLPublicBasicName = "eacl-public-read-write"
+
+	// EACLPrivateBasicName is a well-known name for 0x0C8C8CCC basic ACL.
+	EACLPrivateBasicName = "eacl-private"
+
+	// EACLReadOnlyBasicName is a well-known name for 0x0FBF8CFF basic ACL.
+	EACLReadOnlyBasicName = "eacl-public-read"
+
+	// EACLPublicAppendName is a well-known name for 0x0FBF9FFF basic ACL.
+	EACLPublicAppendName = "eacl-public-append"
+)
+
+// ParseBasicACL parse string ACL (well-known names or hex representation).
+func ParseBasicACL(basicACL string) (uint32, error) {
```

Comment:
It should return `BasicACL` type, I think.

Reply:
I thought about it but it can break some code (in sdk we can fix this but in app that use sdk we can not) that use these constants. Or we should start use `BasicACL` everywhere in sdk (e.g. container options)?

Reply:
> Or we should start use BasicACL everywhere in sdk (e.g. container options) and it's ok to have incompatible changes with the next release?

Before v1.0.0 we are ready for incompatible changes (for example, client package awaits for [unified parameter structures](https://github.com/nspcc-dev/neofs-sdk-go/issues/92), this is quite incompatible).

We either use `uint32` or `BasicACL` type everywhere. Latter sounds a bit more appealing to me (opportunity to define [helper functions](https://github.com/nspcc-dev/neofs-node/blob/master/pkg/services/object/acl/basic_helper.go) on this type in SDK). But I don't mind having `uint32` everywhere like we do now, then separate type definition is redundant.